pypit/ginga.py

```diff
@@ -10,6 +10,7 @@
 
 # CANNOT LOAD DEBUGGER AS THIS MODULE IS CALLED BY ARDEBUG
 #from pypit import ardebug as debugger
+import pdb
```

Comment:
I presume this is for a good reason, and I have no problems with it, but maybe we should keep it consistent, and do:

`import pdb as debugger`